nixos/modules/services/web-apps/pihole-web.nix

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
Perhaps we could have an option like: `openFirewall`? Many services are already using that.